src/server/db/db_helper.py

```diff
@@ -438,9 +438,15 @@ async def validate_sql(self, db_name, query):
             # return False
 
 class sqlite_database(Database):
+    def __init__(self):
+        self.temp_db_folder = "temporary_db"
```

Comment:
Dont call the folder as temporary_db.
Make the directory like this. 
There is a folder called files at the root where we store all the sql files. I would suggest create a fodler sqlite_db inside the files folder and then save the db files in that fodler

Reply:
Done